java-checks/src/test/files/checks/PrintfMisuseCheck.java

```diff
@@ -177,6 +178,21 @@ void foo(Calendar c){
     slf4jLog.warn("message ", 1); // Noncompliant {{String contains no format specifiers.}}
     slf4jLog.warn("message {}", 1);
     slf4jLog.warn("message");
+
+    java.util.logging.Logger logger2 = java.util.logging.Logger.getLogger("som.foo", "som.foo.resources.i18n.LogMessages");
```

Comment:
If I'm not wrong, this case would still raise an FP:
```
java.util.logging.Logger logger2;
logger2 = java.util.logging.Logger.getLogger("som.foo", "som.foo.resources.i18n.LogMessages");
logger2.log(java.util.logging.Level.WARNING, "som.foo.errorcode", 404);
```

Reply:
You are right but I would not care too much. We are talking about loggers here, reassigning them seems to me a corner case.

Reply:
Agreed, LGTM then.